### PR TITLE
Update dependencies and better Webpack config

### PR DESCRIPTION
**Pull request by BarthesSimpson**
Pretty similar to the PR for the saga boilerplate.

### CODE REVIEW

File: src/ducks/count.spec.js [hunk 1]
```diff
@@ -4,18 +4,11 @@ import { incrementAsync } from 'ducks/count'
...
-    pathname: 'blank',
-    query: {},
-    search: ''
-  }
-}
+const initialState = store.getState()
 
 describe('incrementAsync', () => {
-  it('dispatches the increment action after one second', async () => {
+  it('Asynchronously increments the counter', async () => {
```
Comment (granmoe) [inline src/ducks/count.spec.js:10]: Good change :-)

File: src/public/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-  	<title>Redux Saga Boilerplate</title>
+  	<title>Redux Async/Await Boilerplate</title>
```
Comment (granmoe) [inline src/public/index.html:4]: Ah, good catch

File: src/store/index.js [hunk 3]
```diff
@@ -35,7 +30,7 @@ export default function () {
 
   const store = createStore(
     combineReducers(allReducers),
-    compose(enhancer, applyMiddleware(...middlewares))
+    composeWithDevTools(enhancer, applyMiddleware(...middlewares))
```
Comment (granmoe) [inline src/store/index.js:33]: Do you prefer dev tools to the logger? I liked the logger, but I'm not married to it (and the code to conditionally add it felt like unnecessary goo).

**Review by granmoe — APPROVED** [review]

File: src/store/index.js (hunk 3 above)
Comment (BarthesSimpson) [reply src/store/index.js:33]: I'm obsessed with redux devtools :) Remind me to show you why this Saturday

Comment (granmoe) [reply src/store/index.js:33]: Will do. I know you can time travel and also tap into them in prod sometimes :-)

**Pull request merged**